server/@types/users/index.d.ts

```diff
@@ -0,0 +1,8 @@
+import type { Caseload } from '@prison-api'
```

Comment:
Is it worth adding this file to `paths` in tsconfig so we can import like the others?

Reply:
Yeah I initially wasn't sure about this as it's only the one class in the file. Thinking about it now though, I _was_ thinking the user types in our codebase are getting a little confusing (when to use each one?) so it'd be good to tidy them up/consolidate them in the near future, which would mean adding more types to that file.

TL;DR, I'll add an alias to pave the way for tidying up users in future.